apps/wildfly.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apps
+
+import (
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
+)
+
+type LoggingProcessorWildflySystem struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+}
+
+func (LoggingProcessorWildflySystem) Type() string {
+	return "wildfly_server"
```

Comment:
Looks like we're naming this because we're not targeting [Audit logs ](https://docs.wildfly.org/26/Admin_Guide.html#Audit_logging). Looks like audit logs are off by default, and have a different structure. Not too sure what naming pattern we should use here. I was thinking maybe having this be `wildfly` and for audit logs we'd have `wildfly_audit`. Will defer to @qingling128's judgement

Reply:
I'm interpreting @qingling128 's comments to rename pipelines to "wildfly_system" as an acceptance of the current naming. Confirm here if that wasn't your intention Ling!

Reply:
Either `wildfly_server` or `wildfly_system` is OK to me.

I do see `server logs` pop up in their documentation here and there. And the current log files say `server.log`.

If we are adding `./domain/log/*.log` though, it's better as `wildfly_system` logs so that it's more generic.